frontend/src/components/Home.vue

```diff
@@ -21,6 +21,50 @@
           <v-toolbar card color="grey lighten-3">
             <v-toolbar-title>Submit Application Configuration</v-toolbar-title>
             <v-spacer></v-spacer>
+
+            <v-tooltip bottom>
+              <template v-slot:activator="{ on }">
+                <v-btn
+                  fab
+                  small
+                  color="primary"
+                  v-on="on"
+                  @click.stop="dialog = true"
+                  @click="getHealthCheck"
+                >
+                  <v-icon>healing</v-icon>
+                </v-btn>
+              </template>
+              <span>API Health Check</span>
+            </v-tooltip>
+
+            <v-dialog v-model="dialog" width="600">
+              <v-card>
+                <v-card-title class="headline primary white--text" primary-title>
+                  <v-icon dark right>healing</v-icon>&nbsp; API Health Check
```

Comment:
I'm presuming there's no alternative to using the `&nbsp;` for this?

Reply:
Adjusting to use a spacer instead